tests/roles/nova_adoption/tasks/wait.yaml

```diff
@@ -0,0 +1,12 @@
+# NOTE(bogdando): Status phase 'Running' doesn't necessarily mean it IS running in fact.
+# Instead, wait for CR Ready status, in short runs (https://github.com/kubernetes/kubectl/issues/1120)
+# of a total 10x6x5=300s waiting
```

Comment:
* We are waiting for Nova CR status not Pod status so I don't think this bug is relevant. Have you see this hanging?
* I think your math is wrong here. If I understand correctly ansible runs the shell command 6 times and waits 5 seconds between each run. As each oc run takes a maximum 10 seconds, the whole waiting time is (5 + 10) x 6 = 90 seconds not 300 seconds

Reply:
indeed, this issue probably have no sense for CRs